vm_supervisor/storage.py

```diff
@@ -155,28 +161,98 @@ def create_ext4(path: Path, size_mib: int) -> bool:
     return True
 
 
+def check_device_exists(path: Path) -> bool:
+    try:
+        return stat.S_ISBLK(os.stat(path).st_mode)
+    except Exception:
+        return False
+
+
+async def create_devmapper(volume: PersistentVolume, namespace: str) -> Path:
```

Comment:
This is a very long function. Should it be split in multiple, easier to understand functions ?

Reply:
I can split, but it doesn't make a lot of sense to do it in my opinion, because the command execution needs to be done in order to work.